tests/features/menus/feature_set.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Feature set menu
+  In order to easily enable a feature
```

Comment:
I have changed the way administrator is granted access to the feature set menu.
Therefore it is part of the scope of this ticket to ensure i have not created a regression in doing this. BUT i copied user menu and did not create the test to replace it, so i need to fix that :D